### ISSUE TITLE
Inheritance issue with HBase 1.0 and Kundera 2.16

### ISSUE THREAD

**Issue by monkegoist** [Impetus/kundera#725]
See `HBaseDataHandler#populateEntityFromHBaseData()`:

``` java
String discrColumn = ((AbstractManagedType) entityType).getDiscriminatorColumn();
String discrValue = ((AbstractManagedType) entityType).getDiscriminatorValue();

if (discrColumn != null && hbaseData.getColumnValue(discrColumn) != null 
    && discrValue != null) {
    ...
}
```

Condition `hbaseData.getColumnValue(discrColumn) != null` will always be false because `hbaseData` contains keys which look like `tableName:columnName` and `discrColumn` is not prefixed by table name.

Prefixing `@DiscriminatorColumn`'s name with table name doesn't work since I end up with something like `column=users:users:_type`.

**Comment by pragalbhgarg** [comment]
@monkegoist 
We will be fixing this bug as soon as possible.

**Comment by pragalbhgarg** [comment]
@monkegoist 
We have fixed issues related to JPA inheritence in kundera-hbase-v2. Please refer these [test cases](https://github.com/impetus-opensource/Kundera/blob/trunk/src/kundera-hbase/kundera-hbase-v2/src/test/java/com/impetus/client/hbase/crud/inheritence/InheritenceDomainTest.java) in the trunk. Let us know if you face other issues!

**Comment by chhavigangwal** [comment]
Fixed with Kundera -2.17 release.
